examples/ecs/fixed_timestep.rs

```diff
@@ -0,0 +1,45 @@
+use bevy::{
+    core::{FixedTimestep, FixedTimesteps},
+    prelude::*,
+};
+
+const LABEL: &str = "my_fixed_timestep";
+
+fn main() {
+    App::build()
+        .add_plugins(DefaultPlugins)
+        // this system will run once every update (it should match your screen's refresh rate)
+        .add_system(update)
+        // add a new stage that runs every two seconds
+        .add_stage(
+            "fixed_update",
+            SystemStage::parallel()
+                .with_run_criteria(
+                    FixedTimestep::step(2.0)
+                        // labels are optional. they provide a way to access the current FixedTimestep state from within a system
+                        .with_label(LABEL),
```

Comment:
Is there a duplication here? We have to invent a name for the Stage and a label for SystemStage.
Can these be unified?

Reply:
I don't think we should unify them:

1. A StateStage has different "substages" for each State lifecycle handler. That means there can be a 1-&gt;Many relationship between stage name and run criteria.
2. Not all stages have names (only stages that live inside a Schedule)
3. The upcoming SystemSets will also have run criteria, and they definitely won't have an infer-able name.